rebar.config

```diff
@@ -178,7 +180,24 @@
    {override, jwerl, [{plugins, [rebar3_elixir, rebar3_hex]}]}
  ]}.
 
-{dialyzer, [{plt_extra_apps, [jid, cowboy, lasse, p1_utils, ranch, gen_fsm_compat, epgsql]}]}.
+{dialyzer, [
```

Comment:
great job, but probably it's better to have it in a separate PR. it's just a wish for the future, no need to make a split of this one.